src/schema/v2/me/artworkRecommendations.ts

```diff
@@ -0,0 +1,82 @@
+import { GraphQLFieldConfig, GraphQLInt } from "graphql"
+import { connectionFromArray } from "graphql-relay"
+import gql from "lib/gql"
+import { convertConnectionArgsToGravityArgs, extractNodes } from "lib/helpers"
+import { compact, find } from "lodash"
+import { CursorPageable, pageable } from "relay-cursor-paging"
+import { createPageCursors } from "schema/v1/fields/pagination"
+import { artworkConnection } from "schema/v2/artwork"
+import { ResolverContext } from "types/graphql"
+
+// This limits the maximum number of artworks we receive from Vortex and is related to how we implement the Connection in this resolver.
+const MAX_ARTWORKS = 50
```

Comment:
We are already accepting a `first` parameter, can we use that? or what's the benefit of using the constant?

Reply:
Vortex doesn't allow a `after` parameter and we would have to work with cursors (see this comment: https://github.com/artsy/metaphysics/pull/3797#discussion_r815860540).

Let me know if you have an idea how to make this work.

Reply:
I guess it's ok then, we can reevaluate later